source/material_model/visco_plastic.cc

```diff
@@ -430,9 +430,9 @@ namespace aspect
                                    "composite viscous flow laws. Prior to yielding, one may select to "
                                    "modify the viscosity to account for viscoelastic effects. Plasticity "
                                    "limits viscous stresses through a Drucker Prager yield criterion. "
-                                   "Note that this material model is based heavily on the "
-                                   "DiffusionDislocation (Bob Myhill), DruckerPrager "
-                                   "(Anne Glerum), and Viscoelastic (John Naliboff) material models. "
+                                   "The implementation of this material model is based heavily on the "
+                                   "`DiffusionDislocation' (Bob Myhill), `DruckerPrager' "
```

Comment:
are the different single quote styles \` and ' intentional?

Reply:
To answer for @bangerth, yes, these apply open and close quotation marks (rather than two closing quotes).